### PR TITLE
Add docker file 

### PR DESCRIPTION
**Pull request by BulatSaif**
Docker file added for easy use in remote vm 

### Test:
```
docker run -it --rm soramitsu/iroha-state-migration-tool:1.2.0 --help
````

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.idea
+.dockerignore
+.gitignore
+.git
+Dockerfile
+README.md
+housekeeping
```
Comment (MBoldyrev) [inline .dockerignore]:
```suggestion
.# ignore everything
*
.*
# except following:
!block_storage.py
!migration_data
!requirements.txt
!schema
!schema_version.py
!state_migration.py```

File: README.md [hunk 2]
```diff
@@ -76,6 +76,25 @@ python3 state_migration.py --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_
 python3 state_migration.py --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_password big_russian --pg_dbname iroha_data --target_schema_version 1.4.8 --block_storage_files /tmp/we_still_use_file_block_store/
 ```
 
+## Docker 
+### Build docker image 
+```shell script
+$ dcoker build -t soramitsu/iroha-state-migration-tool:1.2.0 .
```
Comment (MBoldyrev) [inline README.md]: i think we should not give this tag as example as it will conflict with the one from dockerhub

**Review by MBoldyrev — APPROVED** [review]
thanks!

File: README.md (hunk 2 above)
Comment (BulatSaif) [inline README.md]:
```suggestion
$ dcoker build -t soramitsu/iroha-state-migration-tool:local .
```

File: README.md [hunk 3]
```diff
@@ -76,6 +76,25 @@ python3 state_migration.py --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_
...
+### Example using docker image
+Print help:
+```shell script
+docker run -it --rm soramitsu/iroha-state-migration-tool:1.2.0 --help
+```
+
+Migration from 1.1.1 to 1.2.0:
+```
+docker run -it --rm \
+   soramitsu/iroha-state-migration-tool:1.2.0  --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_password big_russian --pg_dbname iroha_data --target_schema_version 1.1.1 --force_schema_version
```
Comment (BulatSaif) [inline README.md]:
```suggestion
   soramitsu/iroha-state-migration-tool:local  --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_password big_russian --pg_dbname iroha_data --target_schema_version 1.1.1 --force_schema_version
```

File: README.md [hunk 4]
```diff
@@ -76,6 +76,25 @@ python3 state_migration.py --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_
...
+docker run -it --rm -v /tmp/we_still_use_file_block_store:/tmp/we_still_use_file_block_store \
+   soramitsu/iroha-state-migration-tool:1.2.0  --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_password big_russian --pg_dbname iroha_data --target_schema_version 1.2.0 --block_storage_files /tmp/we_still_use_file_block_store/
```
Comment (BulatSaif) [inline README.md]:
```suggestion
   soramitsu/iroha-state-migration-tool:local  --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_password big_russian --pg_dbname iroha_data --target_schema_version 1.2.0 --block_storage_files /tmp/we_still_use_file_block_store/
```

File: README.md [hunk 5]
```diff
@@ -76,6 +76,25 @@ python3 state_migration.py --pg_ip 127.0.0.1 --pg_port 5432 --pg_user boss --pg_
...
+## Docker 
+### Build docker image 
+```shell script
+$ dcoker build -t soramitsu/iroha-state-migration-tool:local .
+```
+
+### Example using docker image
+Print help:
+```shell script
+docker run -it --rm soramitsu/iroha-state-migration-tool:1.2.0 --help
```
Comment (BulatSaif) [inline README.md]:
```suggestion
docker run -it --rm soramitsu/iroha-state-migration-tool:local --help
```

**Review by kogeler — APPROVED** [review]
Ok

**Review by Cre-eD — APPROVED** [review]

**Pull request merged**